multinet/errors.py

```diff
@@ -157,7 +157,7 @@ class RequiredParamsMissing(ServerError):
 
     def __init__(self, missing: List[str]):
         """Initialize the exception."""
-        self.missing = missing
+        self.missing = {"missing": missing}
```

Comment:
I am curious, where did you run into this causing a problem? (Just surprised we never ran into it before.)

Reply:
`RequiredParamsMissing` is only used once in `master` [here](https://github.com/multinet-app/multinet-server/blob/master/multinet/api.py#L274), I'm guessing that's why it wasn't noticed. It happens as a result of the exception returning a `list` to Flask, since Flask doesn't allow lists to be returned as endpoint responses unless they are converted to JSON with `jsonify`.

This is the error that currently occurs in `master` if that endpoint I linked above is called without an `edge_table` parameter -
`TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a list.`